v2/stacks/auth/core-ons.yml

```diff
@@ -10,9 +10,93 @@ services:
     extends:
       file: ${PATH_MANIFESTS}/core-ons/dp-api-router.yml
       service: dp-api-router
-  florence: # note: this might not build :( manually delete florence/dist florence/src/node_modules florence/assets/data.go 
+  florence: # note: this might not build :( manually delete florence/dist florence/src/node_modules florence/assets/data.go
     extends:
       file: ${PATH_MANIFESTS}/core-ons/florence.yml
       service: florence
     environment:
-      ENABLE_NEW_SIGN_IN:     "true"
+      ENABLE_NEW_SIGN_IN: "true"
+  dp-frontend-router:
+    extends:
+      file: ${PATH_MANIFESTS}/core-ons/dp-frontend-router.yml
+      service: dp-frontend-router
+  dp-frontend-homepage-controller:
+    extends:
+      file: ${PATH_MANIFESTS}/core-ons/dp-frontend-homepage-controller.yml
+      service: dp-frontend-homepage-controller
```

Comment:
Do you need the homepage-controller to do publishing?

Reply:
This is now addressed.